.github/workflows/validate-release.yml

```diff
@@ -45,6 +45,9 @@ jobs:
     steps:
       - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 #v2.4.0
 
+      - name: Install goimports
+        run: go get golang.org/x/tools/cmd/goimports
```

Comment:
ok, now i see this:
if you do this you add this tool as a dependency in the project and then the check we do for the go modules fail.

please remove this and the job will pass

Reply:
thank you, I've removed these lines.